pkg/server/server.go

```diff
@@ -372,6 +372,10 @@ func (s *Server) ListObjects(ctx context.Context, req *openfgav1.ListObjectsRequ
 	)
 	if err != nil {
 		if errors.Is(err, condition.ErrEvaluationFailed) {
+			var evalError *condition.EvaluationError
+			if errors.As(err, &evalError) {
```

Comment:
Why is this indirection necessary? Just pipe it straight through `ValidationError` as is. No need for the error cast.

Reply:
@jon-whit it boils down to this https://github.com/natefinch/wrap/blob/main/with.go#L108-L120

we'd like this:

```
{
	"code": "validation_error",
	"message": "failed to evaluate relationship condition 'condition2': context is missing parameters '[param1]'"
}
```

vs

```
{
	"code": "validation_error",
	"message": "failed to evaluate relationship condition: failed to evaluate relationship condition 'condition2': context is missing parameters '[param1]'"
}
```

Reply:
Gotcha, thanks for the clarification.